test_files/variables/variables.ts

```diff
@@ -1,2 +1,4 @@
 var v1: string;
 var v2: string, v3: number;
+
+const inferred = {a: 1, b: [{c: '2'}]};
```

Comment:
Can you add a comment to this about what property we're looking for in the output?  (I know we haven't done this in the past, but I have been trying to do this on new tests for future sanity.)

Reply:
Done.